app/models/keyword.rb

```diff
@@ -0,0 +1,24 @@
+require 'lmi_client'
+
+class Keyword < ActiveRecord::Base
+  has_many :occupations
+
+  def add_new_occupations
+    occupations = soc_codes.map do |soc_code|
+      soc_occupation = SocOccupation.find_or_import_from_lmi(soc_code)
+      Occupation.new(
+        soc_occupation: soc_occupation
+      )
+    end
+    self.occupations = occupations
```

Comment:
this will overwrite all occupations on an existing Keyword (which doesn't matter now, but is unexpected), perhaps:
```ruby
self.occupations += occupations
```